doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -62,7 +62,10 @@ You also need the following:
             .\JLink_Windows_V794i_x86_64.exe -InstUSBDriver=1
 
       #. Follow the on-screen instructions.
-      #. After installing, add the J-Link executable to the system path on Linux and MacOS, or to the environment variables on Windows, to run it from anywhere on the system.
+      #. After installing, ensure the J-Link executable can be run from anywhere on your system:
```

Comment:
just a nit, but I think we can remove full stops in lines  47-49, and after the note 70-72, as these are listed items rather than proper sentences.